src/components/OpenWith/OpenWith.js

```diff
@@ -244,6 +308,9 @@ class OpenWith extends PureComponent<Props, State> {
                             integrations={integrations}
                         />
                     )}
+                    {executePostData && (
+                        <ExecuteForm formData={executePostData} id={this.id} />
```

Comment:
It looks like `ExecuteAPI`, `executePostData`, and `formData` all represent the same data. Should the names be consolidated?

Reply:
Fair enough. You'll often see the pattern of `const myThing = new MyThing()` when instantiating classes. Most languages handle type declaration within classes. I figured it might make sense here as well, but it's up to you.